components/builder-web/app/origin/origin-page/origin-members-tab/origin-members-tab.component.ts

```diff
@@ -73,6 +77,10 @@ export class OriginMembersTabComponent implements OnInit, OnDestroy {
     return this.store.getState().origins.currentPendingInvitations;
   }
 
+  get isOriginOwner() {
+    return this.store.getState().users.current.profile.id === this.store.getState().origins.current.owner_id;
+  }
+
```

Comment:
This is the logic i'm using to decide whether or not to show the depart from origin button.